src/main/resources/application.properties

```diff
@@ -25,9 +25,9 @@ spring.flyway.baseline-version=0
 
 springdoc.api-docs.path=/swagger-ui
 
-minio.url=
-minio.accessKey=
-minio.secretKey=
+minio.url=http://thedb.duckdns.org:9000
+minio.accessKey=l9mXzvUAlUV62mRYiYQU
+minio.secretKey=gUEaKClczlPHoFsr4MIXcNcZk1P9jn4osbLCOWuW
```

Comment:
Secret keys don't have a place here.

Reply:
For the GitHub action for build and test to be successful, you can use some invalid keys. It's enough that the parameter is not empty.
For example: 
minio.url=http://duckdns.org
minio.accessKey=1
minio.secretKey=1

Reply:
That's correct. They'll only be required for the minio stuff. Cheers